providers/ionoscloud/dbaas_mongo_cluster.go

```diff
@@ -0,0 +1,42 @@
+package ionoscloud
+
+import (
+	"context"
+	"github.com/GoogleCloudPlatform/terraformer/providers/ionoscloud/helpers"
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	"log"
+)
+``
+type DBaaSMongoClusterGenerator struct {
+	Service
+}
+
+func (g *DBaaSMongoClusterGenerator) InitResources() error {
+	client := g.generateClient()
+	dbaasMongoClient := client.DBaaSMongoApiClient
+	resourceType := "ionoscloud_mongo_cluster"
+
+	response, _, err := dbaasMongoClient.ClustersApi.ClustersGet(context.TODO()).Execute()
+	if err != nil {
+		return err
+	}
+	if response.Items == nil {
+		log.Printf("[WARNING] expected a response containing Mongo DB clusters but received 'nil' instead")
+	}
+	clusters := *response.Items
+	for _, cluster := range clusters {
+		if cluster.Properties == nil || cluster.Properties.DisplayName == nil {
+			log.Printf("[WARNING] 'nil' values in the response for Mongo DB cluster with ID: %v, skipping search for this resource", *cluster.Id)
+			continue
+		}
+		g.Resources = append(g.Resources, terraformutils.NewResource(
+			*cluster.Id,
+			*cluster.Properties.DisplayName+"-"+*cluster.Id,
+			resourceType,
+			helpers.Ionos,
+			map[string]string{},
```

Comment:
I think this needs to be map[string]string{helpers.DcId: *datacenter.Id}, if the cluster references a datacenter

Reply:
@cristiGuranIonos if the cluster references a datacenter, the information will be present in the `connections` object as presented below:

```
resource "ionoscloud_mongo_cluster" "tfer--example_mongo_cluster-03c5ae40-e777-4768-8ac1-d587b2c3a214" {
  connections {
    cidr_list     = ["192.168.1.108/24"]
    datacenter_id = "94a50808-9312-423d-adb6-2142b7b607b5"
    lan_id        = "1"
  }
  .....
}
```